test/extensions/quic_listeners/quiche/platform/quic_epoll_clock.cc

```diff
@@ -0,0 +1,40 @@
+// NOLINT(namespace-envoy)
+//
+// This file is part of the QUICHE platform implementation, and is not to be
+// consumed or referenced directly by other Envoy code. It serves purely as a
+// porting layer for QUICHE.
+
+#include "test/extensions/quic_listeners/quiche/platform/quic_epoll_clock.h"
+
+namespace quic {
+
+bool quic_monotonic_epoll_clock = false;
```

Comment:
Seems like this variable is not used, remove it?

Reply:
done